build.sc

```diff
@@ -0,0 +1,122 @@
+import mill._
+import scalalib._
+import mill.scalalib.publish._
+import scala.util.Try
+
+object lsp extends MavenModule with PublishModule {
+
+  def millSourcePath: os.Path = os.pwd
+
+  def ivyDeps = Agg(
+    ivy"org.eclipse.lsp4j:org.eclipse.lsp4j:0.9.0",
+    ivy"software.amazon.smithy:smithy-model:1.14.1",
+    ivy"io.get-coursier:interface:1.0.4"
+  )
+
+  def publishVersion = T { gitVersion() }
+
+  override def artifactName = s"smithy-language-server"
+
+  def pomSettings = PomSettings(
+    description = "LSP implementation for smithy",
+    organization = "com.disneystreaming.smithy",
+    url = "https://github.com/disneystreaming/smithy-language-server",
+    licenses = Seq(License.`Apache-2.0`),
+    versionControl = VersionControl(
+      Some("https://github.com/disneystreaming/smithy-language-server")
+    ),
+    Seq(Developer("baccata", "Olivier Mélois", "https://github.com/baccata"))
+  )
+
+  def gitVersion: T[String] = T.input {
```

Comment:
Can https://github.com/lefou/mill-vcs-version be used instead?

Reply:
I'll happily switch as soon as Tobias adds MIMA on mill (which in theory is the next big item for him)